pom.xml

```diff
@@ -199,16 +442,16 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
-        <version>3.7.0</version>
+        <version>3.13.0</version>
         <configuration>
-          <source>1.8</source>
-          <target>1.8</target>
+          <source>21</source>
+          <target>21</target>
```

Comment:
We don't want to force 21. I'm happy with moving to Java 11 instead of 8 but I don't want to force all users to move to Java 21. The code should work on >=11.

Reply:
Done.